### DISCUSSION TITLE
Import tasks

### DISCUSSION THREAD

**Discussion by FFaranna** [Leantime/leantime#1955] [Ideas]
it would be very helpful to have the ability to import tasks list, as they can be created very fast via spredsheets. This is possible for expample in tools like shotgrid or kitsu (mainly film/animation tools).

![image](https://github.com/Leantime/leantime/assets/7949710/bbce6eae-9fbe-4e03-974e-0c6da00c49b0)

**Comment by gloriafolaron** [comment]
This is actually in the works with our csv importer. We are still getting through some testing and UI updates for that.

**Reply by rockygiglio** [reply]
when is this coming to the UI....
